pythoncz/models/github.py

```diff
@@ -114,6 +114,7 @@ def _format_issue(org_name, repository, issue, is_pull_request=False):
         'votes': _calculate_votes(issue),
         'labels': labels,
         'coach': 'coach' in labels,
+        'sprint-idea': 'sprint-idea' in labels,
```

Comment:
There is no test for this 😢 (There is test for `coach`) But that's something you can add in a follow-up PR.